ros/src/computing/planning/decision/packages/decision_maker/nodes/decision_maker/decision_maker_node_callback.cpp

```diff
@@ -19,120 +19,54 @@
 
 namespace decision_maker
 {
-void DecisionMakerNode::callbackFromCurrentPose(const geometry_msgs::PoseStamped &msg)
+void DecisionMakerNode::callbackFromFilteredPoints(const sensor_msgs::PointCloud2::ConstPtr& msg)
 {
-  geometry_msgs::PoseStamped _pose = current_pose_ = msg;
-  bool initLocalizationFlag = ctx->isCurrentState(state_machine::INITIAL_LOCATEVEHICLE_STATE);
-  if (initLocalizationFlag &&
-      isLocalizationConvergence(_pose.pose.position.x, _pose.pose.position.y, _pose.pose.position.z,
-                                _pose.pose.orientation.x, _pose.pose.orientation.y, _pose.pose.orientation.z))
-  {
-    ROS_INFO("Localization was convergence");
-  }
-}
+  setEventFlag("received_pointcloud_for_NDT", true);
 
-bool DecisionMakerNode::handleStateCmd(const uint64_t _state_num)
-{
-  bool _ret;
-  ctx->setEnableForceSetState(true);
-  if (!ctx->isCurrentState(_state_num))
-  {
-    _ret = ctx->setCurrentState((state_machine::StateFlags)_state_num);
-    if(_state_num == state_machine::DRIVE_BEHAVIOR_TRAFFICLIGHT_RED_STATE 
-		    || _state_num == state_machine::DRIVE_BEHAVIOR_TRAFFICLIGHT_GREEN_STATE){
-	    isManualLight = true;
-    }
-  }
-  else
-  {
-    _ret = ctx->disableCurrentState((state_machine::StateFlags)_state_num);
-    if(_state_num == state_machine::DRIVE_BEHAVIOR_TRAFFICLIGHT_RED_STATE 
-		    || _state_num == state_machine::DRIVE_BEHAVIOR_TRAFFICLIGHT_GREEN_STATE){
-	    isManualLight = false;
-    }
-  }
-  ctx->setEnableForceSetState(false);
-
-
-  return _ret;
+  /* todo */
+  /* create timer for flags reset   */
```

Comment:
Future tasks should be specified in the function of pull request or managed as isue.

Reply:
I removed this because it is not needed.